frontend/src/app/shared/breadcrumb/breadcrumb.component.html

```diff
@@ -0,0 +1,12 @@
+<nav>
```

Comment:
Since this is not the only `<nav>` element on the page, [navigation elements should be labelled](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements#labeling_section_content). I think you'll have to add a label in the navbar as well.

Reply:
The main navbar has already been labelled, so no changes were required in that component. I have added an `aria-label` to this breadcrumb `nav` element.